guides/novu-fcm-rna.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: 'How to send push notifications in an Android app (react native) with FCM and Novu'
+description: 'Learn how to send FCM push notifications in a React native Android app using Novu and FCM'
+---
+
+## Prerequisites
+
+To complete this tutorial, you will need the following:
+
+
+- A [Firebase](https://firebase.google.com/) account
+- A [Novu](https://web.novu.co/) account
+- An [Expo](https://expo.dev) account
+
+Push notifications are notifications that are sent to a user’s devices whether the user is using the app or not. We’ll be using the Firebase Cloud Messaging (FCM) integration of Novu to send these notifications. You can find the complete code for this project in our [Github repo.](https://github.com/novuhq/fcm-react-native-android) 
+
+## Start a React-Native project
+We'll use the `create-expo` tool to initialize a new expo app. It creates a new React Native app with the expo package installed. We're naming our app `fcm-novu-rn2` and hence using the following command:
+``` javascript
+ npx create-expo-app fcm-novu-rn2
+```
+<Info>If it asks you to choose a template, choose the 'blank' template.</Info>
+
+Now, go to the project directory and install the `react-native-firebase` package using the following command:
+
+``` bash
+npm install --save @react-native-firebase/app
+```
+Once this is installed, install the messaging module of Firebase using the following command:
+``` bash
+npm i --save @react-native-firebase/messaging
+```
+
+## Getting the FCM token
+Now, we need to grab the `token key` from FCM. We'll need this to send notifications using FCM. We're using `useEffect` to grab this key as follows:
+``` jsx
+// Request permission to send notifications
+const authStatus = await messaging().requestPermission();
+
+// Check if permission is granted or provisional
+const enabled =
+  authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+  authStatus === messaging.AuthorizationStatus.PROVISIONAL;
+
+// If permission is granted or provisional, proceed
+if (enabled) {
+  // Log the authorization status
+  console.log('Authorization status:', authStatus);
+
+  // Retrieve the FCM token for the device
+  const token = await messaging().getToken();
+  // Log the FCM token
+  console.log('FCM Token is:', token);
+
+  // Return a function to unsubscribe from notifications
+  return unsubscribe;
+} else {
+  // If permission is not granted, log the error status
+  console.log('Could not grab token...', authStatus);
+}
```

Comment:
Is there anything that needs to be imported here? Where does `messaging()` come from?